scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -852,22 +827,38 @@ def __get_base_attributes( self ):
 
         return attributes
 
-    def __get_log_config_for_container( self, cid, info, k8s_data, base_attributes ):
+    def __get_log_config_for_container( self, cid, info, k8s_cache, base_attributes ):
         result = None
 
         container_attributes = base_attributes.copy()
         container_attributes['containerName'] = info['name']
         container_attributes['containerId'] = cid
         parser = 'docker'
+        common_annotations = {}
+        container_annotations = {}
+        pod_name = '--'
+        pod_namespace = '--'
```

Comment:
This is used as a marker of a clearly invalid name.  It is for the case where a log message might be generated that uses the pod name and namespace, but no pod name or namespace exists (because of an error querying the API).  I have added a comment to this effect in the source.